### PR TITLE
Install jq in the pc_tools and kubectl as a container

### PR DESCRIPTION
**Pull request by ilausuch**
Some queries to cloud providers APIs return JSON results.
jq is a useful command to query these json and check the results

Additionally access to kubernetes clusters require kubectl command.
This is used as a container (same than terraform)

- Related ticket: https://progress.opensuse.org/issues/97715
- Verification run: https://openqa.suse.de/tests/7137054

### CODE REVIEW

**Review by cfconrad — APPROVED** [review]

**Review by jlausuch — APPROVED** [review]

**Comment by ilausuch** [comment]
Now there are 2 commits, one is covering the jq command and the other covers the kubectl

**Review by asmorodskyi — APPROVED** [review]
LGTM , just please update PR title

**Review by jlausuch — APPROVED** [review]
LGTM. Will merge after successful VR.

File: tests/publiccloud/prepare_tools.pm [hunk 1]
```diff
@@ -123,6 +123,16 @@ EOT
     create_script_file('terraform', '/usr/bin/terraform', $terraform_wrapper);
     record_info('Terraform', script_output('terraform -version'));
 
+    # Kubectl in a container
+    my $kubectl_version = 'latest';
```
Comment (b10n1k) [inline tests/publiccloud/prepare_tools.pm:127]: isnt it better to use a specific version. there are a lot of post online. i attach one random https://vsupalov.com/docker-latest-tag/#:~:text=You%20should%20avoid%20using%20the,apart%20from%20the%20image%20ID.

Comment (ilausuch) [reply tests/publiccloud/prepare_tools.pm:127]: Ok, Changing it

Comment (ilausuch) [reply tests/publiccloud/prepare_tools.pm]: Done

**Comment by jlausuch** [comment]
The command `podman run -v /root/:/root/ --rm bitnami/kubectl:$kubectl_version` fails:
https://openqa.suse.de/tests/7131211#step/prepare_tools/246
`The connection to the server localhost:8080 was refused - did you specify the right host or port?`

**Comment by ilausuch** [comment]
> The command `podman run -v /root/:/root/ --rm bitnami/kubectl:$kubectl_version` fails:
> [openqa.suse.de/tests/7131211#step/prepare_tools/246](https://openqa.suse.de/tests/7131211#step/prepare_tools/246)
> `The connection to the server localhost:8080 was refused - did you specify the right host or port?`

Yes, checked that. I changed the script command a little to avoid this error and check that the output is the expected. I tried with some flags for the kubectl command, but all of them tries to connect to the server.

**Comment by ilausuch** [comment]
New test https://openqa.suse.de/tests/7136132#live

**Comment by b10n1k** [comment]
you may want to update the copyright as well

File: tests/publiccloud/prepare_tools.pm [hunk 2]
```diff
@@ -123,6 +123,16 @@ EOT
...
 
+    # Kubectl in a container
+    my $kubectl_version = '1.22';
+    my $kubectl_wrapper = <<EOT;
+#!/bin/sh
+podman run -v /root/:/root/ --rm bitnami/kubectl:$kubectl_version \$@
+EOT
+
+    create_script_file('kubectl', '/usr/bin/kubectl', $kubectl_wrapper);
+    record_info('kubectl', script_output('kubectl version 2>/dev/null | grep "Client Version"'));
```
Comment (jlausuch) [inline tests/publiccloud/prepare_tools.pm:134]:
```suggestion
    record_info('kubectl', script_output('kubectl version --client=true'));
```

Comment (ilausuch) [reply tests/publiccloud/prepare_tools.pm]: yes done

**Comment by ilausuch** [comment]
> you may want to update the copyright as well

Done

**Pull request merged**